cmd/oras/internal/output/print_test.go

```diff
@@ -43,23 +43,49 @@ func (mw *mockWriter) String() string {
 func TestPrint_Error(t *testing.T) {
 	mockWriter := &mockWriter{}
 	printer := NewPrinter(mockWriter, false)
-	printer.Println("boom")
+	err := printer.Println("boom")
 	if mockWriter.errorCount != 1 {
 		t.Error("Expected one errors actual <" + strconv.Itoa(mockWriter.errorCount) + ">")
 	}
+	if err != nil {
+		t.Error("Expected error to be ignored")
+	}
 }
 
 func TestPrint_NoError(t *testing.T) {
```

Comment:
According to the [testing doc](https://pkg.go.dev/testing#hdr-Examples), this function should be named as:
```suggestion
func TestPrinter_PrintVerbose_noError(t *testing.T) {
```